docs/materials/motion/Motion.stories.mdx

```diff
@@ -66,61 +66,19 @@ Orbit exposes 5 [durations](https://developer.mozilla.org/en-US/docs/Web/CSS/tra
 
 Used in animations that are not in the way, they quickly change from one state to another.
 
+<code>cubic-bezier(0.22, 0.61, 0.36, 1)</code>
+
 ### Focus
 
 Used in animations that are designed to draw the user’s attention on what changed, in order to let them move on with their next task.
 
+<code>cubic-bezier(0.46, 0.03, 0.52, 0.96)</code>
+
 ### Expressive
 
 Used in animations that are meant to give a sense of completeness or resolution to the user. Use them wisely.
 
```

Comment:
It's fine to remove these but you the Motion page should have links to the the tokens page

Reply:
The code changed before I posted the review. I am writing about the removal of the tokens from this page. The page should contains links to the tokens page.

Reply:
No, my comment was on the removed code. But before I publish my review, I think you pushed changes to the PR and it moved the comment there.

Reply:
All good then?

Reply:
Somehow, the Motions page should have a link to the Easing tokens